elfi/core.py

```diff
@@ -511,7 +545,7 @@ def _inherit_observed(self):
             if not hasattr(parent, "observed"):
                 raise ValueError("Parent {} has no observed value to inherit".format(parent))
         observed = tuple([p.observed for p in self.parents])
-        observed = self.operation({"data": observed, "n": 1})["data"]
+        observed = self._transform({"data": observed, "n": 1})["data"]
```

Comment:
Maybe, currently I don't want it to be changed without the core knowing about it.